meta-xt-domu/recipes-graphics/wayland/weston-init.bbappend

```diff
@@ -0,0 +1,8 @@
+FILESEXTRAPATHS:prepend := "${THISDIR}/${PN}:"
+
+do_install:append() {
+    # Disable the idle timeout, although this could be done by using
+    # PACKAGECONFIG, but meta-renesas layer supplies its own weston.ini, so
+    # original file located at poky/meta/recipes-graphics/wayland/ gets skipped.
+    sed -i -e "/^\[core\]/a idle-time=0" ${D}${sysconfdir}/xdg/weston/weston.ini
```

Comment:
We don't want to use `sed` anymore for such tasks, because it breaks very easily. We need another mechanism to do this.

Reply:
ok, makes sense, will use echo in that particular case

Reply:
"+"